services/movie.py

```diff
@@ -0,0 +1,34 @@
+from typing import List, Optional
+
+from django.db.models.query import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(genres_ids: Optional[List[int]] = None,
+               actors_ids: Optional[List[int]] = None) -> QuerySet:
+    result_query = Movie.objects.all()
+
+    if genres_ids:
+        result_query = result_query.filter(genres__id__in=genres_ids)
+
+    if actors_ids:
+        result_query = result_query.filter(actors__id__in=actors_ids)
+
+    return result_query
+
+
+def get_movie_by_id(movie_id: int) -> Optional[Movie]:
+    return Movie.objects.get(id=movie_id)
+
+
+def create_movie(movie_title: str,
+                 movie_description: str,
+                 genres_ids: Optional[List[int]] = None,
+                 actors_ids: Optional[List[int]] = None) -> None:

```

Comment:
Place each argument on a new line so that there would be no arguments on the same line with opening and closing parenthesis. It will help to improve the code style and readability. Apply this change to all functions' signatures